extensions/smallrye-openapi/spi/src/main/java/io/quarkus/smallrye/openapi/deployment/spi/IgnoreStaticDocumentBuildItem.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.smallrye.openapi.deployment.spi;
+
+import java.util.regex.Pattern;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public final class IgnoreStaticDocumentBuildItem extends MultiBuildItem {
```

Comment:
Add a meaningful javadoc so this is documented in https://quarkus.io/guides/all-builditems